calm/control-example/one-node-wonder.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://raw.githubusercontent.com/finos/architecture-as-code/main/calm/draft/2024-08/meta/calm.json",
+  "id": "one-node-wonder.json",
+  "title": "Demonstration of domain and controls",
+  "nodes": [
+    {
+      "unique-id": "example-system",
+      "node-type": "system",
+      "name": "Example System",
+      "description": "Example System",
+      "domains": {
+        "cbom": [
+          {
+            "unique-id": "ci-arch-001",
```

Comment:
Is this the unique ID of the control instance, or of the evidence?

Reply:
@yt-ms calm unique id. This would be for anything that relates to this domain (we don't have that yet)

Reply:
@yt-ms we can go this way, however this would mean that we'd need the core CALM schema to understand the types of the controls domain and future domains that we add. What is there at the moment is represented by the `domain-type`. We could modify the `domain-type` to be like (may not quite be syntax correct):

```
"domain-type": {
      "type": "object",
      "properties": {
        "unique-id": {
          "type": "string"
        },
        "description": {
          "type": "string"
        },
        "detail": {
          "type": "string"
        },
        "evidence": {
          "type": "array",
          "items": {
                  "evidence-id": {
                      "type": "string"
                  },
                 "evidence-link": {
                      "type": "string"
                  },
          }
        }
      },
      "required": [
        "unique-id",
        "description",
        "detail"
      ]
    }
```
Would this be sufficient?